### ISSUE TITLE
filter_relabel

### ISSUE THREAD

**Issue by zh190920** [rtmaww/SENT#16]
why do labels have to be reassigned to their original labels at each iteration in filter_relabel
`train_batch_data = self.train_dl.dataset`
`for i in range(len(train_batch_data)):`
      `train_batch_data[i][-1][:] = self.ori_train_labels[i]`
`self.train_dl = DataLoader(train_batch_data, batch_size=self.batch_size, sampler=SequentialSampler(train_batch_data))`
